tests/scripts/thread-cert/Cert_5_2_05_AddressQuery.py

```diff
@@ -32,126 +32,178 @@
 import command
 import config
 import thread_cert
+from pktverify.consts import WIRESHARK_OVERRIDE_PREFS, MLE_PARENT_REQUEST, MLE_PARENT_RESPONSE, MLE_CHILD_ID_REQUEST, MLE_CHILD_ID_RESPONSE, MLE_LINK_REQUEST, ADDR_SOL_URI, ADDR_NTF_URI, SOURCE_ADDRESS_TLV, MODE_TLV, TIMEOUT_TLV, CHALLENGE_TLV, RESPONSE_TLV, LINK_LAYER_FRAME_COUNTER_TLV, MLE_FRAME_COUNTER_TLV, ROUTE64_TLV, ADDRESS16_TLV, LEADER_DATA_TLV, NETWORK_DATA_TLV, TLV_REQUEST_TLV, SCAN_MASK_TLV, CONNECTIVITY_TLV, LINK_MARGIN_TLV, VERSION_TLV, ADDRESS_REGISTRATION_TLV, NL_MAC_EXTENDED_ADDRESS_TLV, NL_RLOC16_TLV, NL_STATUS_TLV, NL_TARGET_EID_TLV, NL_ML_EID_TLV, COAP_CODE_ACK
+from pktverify.packet_verifier import PacketVerifier
+from pktverify.null_field import nullField
+from pktverify.bytes import Bytes
 
 LEADER = 1
 ROUTER1 = 2
 BR = 3
-ED1 = 17
+MED = 17
 DUT_REED = 18
 ROUTER_SELECTION_JITTER = 1
 
+# Test Purpose and Description:
+# -----------------------------
+# The purpose of this test case is to validate that the DUT is able to generate
+# Address Notification messages in response to Address Query messages.
+#
+#  - Build a topology that has a total of 16 active routers, including the Leader,
+#    with no communication constraints
+#  - The Leader is configured as a DHCPv6 server for prefix 2001::
+#  - The Border Router is configured as a SLAAC server for prefix 2002::
+#
+# Test Topology:
+# -------------
+#               MED
+#                |
+#  Router_15 - Leader
+#      ...    /     \
+#         Router_2  Router_1
+#           [BR]       |
+#                   REED(DUT)
+#
+# DUT Types:
+# ----------
+#  REED
+
 
 class Cert_5_2_5_AddressQuery(thread_cert.TestCase):
+
```

Comment:
more clear, agree

Reply:
since DUT only allows Router1, no need Leader denies DUT ? can we change to :
Build a topology that has a total of 16 active routers, including the Leader, with no communication constraints and
MED only allows Leader
DUT only allows Router1
DUT allows BR later as required in step 5.

Reply:
sg